### DISCUSSION TITLE
Notification on chat message received

### DISCUSSION THREAD

**Discussion by myxlot** [inspireui/support#5632] [Working on Features]
At this time notifications are send on an order status change, but not when a customer contacts the vendor through the messenger. This means that currently the customer/vendor need to log into the app & go to the messenger to check if there is any new communication.

This 'polling' is pretty cumbersome nowadays & hard to explain to users when all other apps with messaging in place will give them a notification upon receiving new messages.

**Comment by pasarsegar** [comment]
Customer and vendor chat notification NEEDED!

**Comment by simonpham** [comment]
Push notification for Firebase Realtime chat is available in FluxStore 3.4.0.

**Reply by Jossua-demavie** [reply]
but there is a problem. I have the 3 applications (Fluxstore Manager, Fluxstore Multi-vendor and Fluxstore delivery boy). I followed the documentation but the automatic notifications do not work. I even contacted support who told me that they could not do anything for the moment
